schema.sql

```diff
@@ -1,3 +1,19 @@
+-----ANIMALS TABLE-----
 CREATE TABLE animals ( ID int, Name varchar(100), Date_of_birth date, Escape_attempts int, Neutered boolean, Weight_kg decimal );
+
+ALTER TABLE animals DROP COLUMN ID; -----> deletes a specific column(ID)
+ALTER TABLE animals ADD ID SERIAL PRIMARY KEY;
+
 ALTER TABLE animals ADD Species varchar(100);
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id varchar(100);
+
+ALTER TABLE animals ADD COLUMN owner_id varchar(100);
```

Comment:
- [ ] The `species_id` and `owner_id` are foreign keys for the `species` table and `owenrs` table respectively. Therefore, you should use a syntax like the following. You may also check [this](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-foreign-key/) page for further reading.
```
ALTER TABLE child_table 
ADD CONSTRAINT constraint_name 
FOREIGN KEY (fk_columns) 
REFERENCES parent_table (parent_key_columns);
```